dm/config/security_test.go

```diff
@@ -171,3 +171,21 @@ func (c *testTLSConfig) TestClone() {
 	clone.CertAllowedCN[0] = "g"
 	c.Require().NotEqual(s, clone)
 }
+
+func (c *testTLSConfig) TestWriteTLSContentToFiles() {
```

Comment:
can you write new test with `"github.com/stretchr/testify/require"`, we ever planned to replace all those old ones with this, but only half done

Reply:
fixed